### PR TITLE
Remove flag in language selection

### PR DESCRIPTION
**Pull request by anowlet**
#### [Flags are not languages](http://www.flagsarenotlanguages.com/blog/)
A blog about designing global user experiences: beyond language, location & culture.

### CODE REVIEW

**Comment by alexandrebodin** [comment]
Hello @anowlet thanks you for contributing !

Can you please add a short description so it is easy to understand what the PR is about ? 

The article makes a lot of sense I think we need to consider removing the flag indeed :) We will check we our designer if we need ot change sth before merging this PR ;)

**Comment by alexandrebodin** [comment]
FYI design screenshot for design review. 
Screenshot 2020-07-22 at 10 22 18

**Comment by Aurelsicoko** [comment]
The acronym should be right-aligned, is it possible to update the PR then it looks good to me ;)

**Comment by alexandrebodin** [comment]
@anowlet Any way you can make the update ?

**Comment by alexandrebodin** [comment]
Closing this PR for inactivity. Feel free to reopen if you get the time to update :)

**Comment by anowlet** [comment]
http://www.flagsarenotlanguages.com/blog/why-flags-do-not-represent-language/

Flags are unique to a country or nation: but languages are often spoken across national borders. By using a flag for a language, you may **confuse** or even **offend** users.

**Comment by alexandrebodin** [comment]
@anowlet  we totally agree we just need the PR to be updated according to the review (design update) before merging it. If you can do it we will reopen and merge 👍

**Comment by anowlet** [comment]
image
I think it should show the native name.

File: docs/v3.x/admin-panel/customization.md [hunk 1]
```diff
@@ -61,6 +61,11 @@ const trads = {
   fr,
 };
 
+export const languageNativeNames = {
+  en: 'English',
+  fr: 'français',
```
Comment (soupette) [inline docs/v3.x/admin-panel/customization.md:66]: `fr: 'Français',`

File: packages/strapi-admin/admin/src/translations/index.js [hunk 2]
```diff
@@ -44,4 +44,28 @@ const trads = {
   zh,
 };
 
+export const languageNativeNames = {
+  ar: 'العربية',
+  cs: 'čeština',
+  de: 'Deutsch',
+  en: 'English',
+  es: 'español',
```
Comment (soupette) [inline packages/strapi-admin/admin/src/translations/index.js:52]: Can you uppercase the first letter of the native language name when possible?

Comment (anowlet) [reply packages/strapi-admin/admin/src/translations/index.js:52]: https://www.google.com.hk/preferences#languages 
I referred to this translation.  Not all regions need to be uppercased

Comment (anowlet) [reply packages/strapi-admin/admin/src/translations/index.js:52]: Jietu20200812-105031

**Review by alexandrebodin — APPROVED** [review]

**Comment by alexandrebodin** [comment]
@anowlet Thank you for this improvement 💯

**Pull request merged**